R/g.analyse.perday.R

```diff
@@ -320,23 +324,50 @@ g.analyse.perday = function(ndays, firstmidnighti, time, nfeatures,
             for (mi in 2:ncol(metashort)) { #run through metrics (for features based on single metrics)
```

Comment:
It seems vari2 calculations do not need to be part of the of this loop, because for every iteration of the loop they are identical. So, I think they need to be moved up outside the loop?